plugins/inventory/proxmox.py

```diff
@@ -528,10 +543,10 @@ def _populate_pool_groups(self, added_hosts):
     def _populate(self):
 
         # create common groups
-        self.inventory.add_group(self._group('all_lxc'))
-        self.inventory.add_group(self._group('all_qemu'))
-        self.inventory.add_group(self._group('all_running'))
-        self.inventory.add_group(self._group('all_stopped'))
+        default_groups = ['lxc', 'qemu', 'running', 'stopped', 'prelaunch', 'paused']
```

Comment:
I tihnk groups for the new statuses should only be created if `qemu_extended_statuses` is `True`.

Reply:
Changed it so they are created only `qemu_extended_statuses` is `True`